sklearn/naive_bayes.py

```diff
@@ -579,7 +579,9 @@ class MultinomialNB(BaseDiscreteNB):
     ----------
     C.D. Manning, P. Raghavan and H. Schuetze (2008). Introduction to
     Information Retrieval. Cambridge University Press, pp. 234-265.
-    http://nlp.stanford.edu/IR-book/html/htmledition/naive-bayes-text-classification-1.html
+
+    http://nlp.stanford.edu/IR-book/html/htmledition/\
```

Comment:
PS: this has nothing to do with numpydoc. Numpydoc concerns the formatting of sections like Parameters with:
```
identifier : type-spec
    description
```
and how to turn that into sphinx.